cip/CIP2016-07-07-Parameter-syntax.adoc

```diff
@@ -0,0 +1,91 @@
+= CIP2016-07-07 - Parameter syntax
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+This CIP suggests changing syntax for parameter usage, from curly braces (`{}`) to dollars (`$`).
+Curly brackets are used by the language for literal maps and property predicates, and this overloading may cause confusion in usage.
```

Comment:
I suggest the following sentence instead (or something similar):
'As curly braces are already used by the language for literal maps.... predicates, their usage as parameter delimiters may lead to confusion when writing or reading queries.'